quickstart/README.adoc

```diff
@@ -0,0 +1,107 @@
+:imagesdir: ../images
+
+== Starting the Server
+
+=== Docker
+
+To start and bind to localhost:8080
+
+[source,shell]
+----
+docker run -p 8080:8080 jboss/hawkular-apm-server-dev
+----
+
+This docker image starts a server with preconfigured user `jdoe` and password `password`.
+
+=== Binary Release
+
+Download and unpack the latest `hawkular-apm-dist` distribution from https://github.com/hawkular/hawkular-apm/releases[Hawkular APM releases].
+
+From the distribution's root folder:
+
+[source,shell]
+----
+bin/add-user.sh -a -u jdoe -p password -g read-write,read-only
+bin/standalone.sh
+----
+
+
+=== Source Build
+
+[source,shell]
+----
+git clone https://github.com/hawkular/hawkular-apm.git
+cd hawkular-apm
+mvn clean install -Pdev  // <1>
+cd dist/target/hawkular-apm-dist-${version}
+bin/standalone.sh
```

Comment:
We should provide also `-b 0.0.0.0` and` -Djboss.http.port`. Someone may already be using zipkin somewhere in the cloud without binding to all addresses it would not see APM server..